src/app/core/components/expansion-panel/expansion-panel.component.html

```diff
@@ -0,0 +1,43 @@
+<mat-accordion>
+  <mat-expansion-panel (opened)="panelOpenState = true"
+                       (closed)="panelOpenState = false">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        {{ element.name }}
+      </mat-panel-title>
+      <mat-panel-description>
+        <span *ngIf="element.id">ID: {{ element.id }}</span>
+      </mat-panel-description>
+    </mat-expansion-panel-header>
+    <app-general-styles *ngIf="element.name === 'general styles'"></app-general-styles>
+    <app-form-style-input *ngIf="element.name === 'input'"
```

Comment:
use ENUM instead of string in places like this
```suggestion
    <app-form-style-input *ngIf="element.name === ENUM"
```